Automattic/wp-calypso
# Untangling: revert refactoring of Settings -> General -> Site Tools

Reverts https://github.com/Automattic/wp-calypso/pull/87123

## Proposed Changes

This PR reverts the extraction of Site Tools from Settings -> General. It is now an abandoned idea from the past Untangling Calypso project. The added route (`/settings/site-tools`) never made it to production.

Cleaning up this code now because otherwise it has been a friction when untangling the Site Settings page.

## Why are these changes being made?

Code cleanup.

## Testing Instructions

1. Go to `/settings/general/:siteSlug?flags=-untangling/hosting-menu` of a Default site.
    - Verify it still works.
1. Go to `/settings/general/:siteSlug?flags=-untangling/hosting-menu` of a Classic site.
    - Verify it still works.

## Pre-merge Checklist
